test-py/test_backends_designspace.py

```diff
@@ -222,6 +222,28 @@ async def test_addAnchor(writableTestFont):
     )
 
 
+async def test_read_glyph_locked(writableTestFont):
+    glyphName = "space"
+    glyph = await writableTestFont.getGlyph(glyphName)
+
+    assert glyph.customData.get("fontra.glyph.locked") is True
+
+
+async def test_add_glyph_locked(writableTestFont):
```

Comment:
Maybe rename to `test_write_glyph_locked`

Reply:
done